src/main/java/Wolox/training/models/Book.java

```diff
@@ -74,6 +76,7 @@ public String getTitle() {
     }
 
     public void setTitle(String title) {
+        checkNotNull(title);
```

Comment:
It is good practice to set up a message to indicate which field fails the check. The message should be a constant.
It is worth mentioning that, when dealing with required strings, you should also check that the string is not empty. To do this, you can use Guava's `checkArgument` and `Strings.isNullOrEmpty()` method.
```suggestion
        checkArgument(!Strings.isNullOrEmpty(title), TITLE_FIELD_MESSAGE);
```